opensquirrel/exporter/quantify_scheduler_exporter.py

```diff
@@ -17,14 +17,13 @@
 if TYPE_CHECKING:
     from opensquirrel.circuit import Circuit
 
-
 # Radian to degree conversion outcome precision
 FIXED_POINT_DEG_PRECISION = 5
 
 
 class _ScheduleCreator(IRVisitor):
-    def _get_qubit_string(self, q: Qubit) -> str:
-        return f"{self.qubit_register_name}[{q.index}]"
+    def _get_qubit_string(self, q: QubitLike) -> str:
```

Comment:
I would say this is yet another case where we want to restrict `q` to being a `Qubit`. This is an internal function. If you have a look at the points where this function is called, you'll see we always call it with a qubit.

So we can get rid of all the `Qubit()` conversions.